fastparquet/writer.py

```diff
@@ -137,7 +140,10 @@ def convert(data, se):
         out = data.values
     elif dtype == "O":
         if converted_type == parquet_thrift.ConvertedType.UTF8:
-            out = array_encode_utf8(data)
+            if PY2:
+                out = np.array([x.encode('utf8') for x in data], dtype="O")
```

Comment:
Ideally `array_encode_utf8` should be fixed instead, though I don't have a problem personally if Python 2 takes a backseat in terms of performance :-)